specification/communication/data-plane/Rooms/preview/validate_log_20220413_221737.log

```diff
@@ -0,0 +1,23 @@
+2022-04-14T05:17:37.968Z - silly: Initializing OpenApiDiff class
```

Comment:
do we need this file?

Reply:
Not required. Removed validate log file from the PR.